shoop/front/middleware.py

```diff
@@ -88,6 +90,11 @@ def _set_timezone(self, request):
     def process_response(self, request, response):
         if hasattr(request, "basket") and request.basket.dirty:
             request.basket.save()
+
+        is_superuser = request.user and request.user.is_superuser
```

Comment:
You'll want to do this during `process_request`, otherwise the pages do all their work and then end up showing a "not available" page :\